backend/ligand_loader.py

```
@@ -0,0 +1,966 @@
+except ImportError:
+    PUBCHEM_AVAILABLE = False
+
+try:
+    from chembl_webresource_client.new_client import new_client
+    CHEMBL_AVAILABLE = True
+except ImportError:
+    CHEMBL_AVAILABLE = False
+    new_client = None
+
+
+def test_chembl_connectivity(timeout: int = 5) -> bool:
```

Ruff (0.15.2)

[warning] Unused function argument: `timeout`

(ARG001)